contracts/InterestRate/StableRateModel.sol

```diff
@@ -0,0 +1,129 @@
+pragma solidity ^0.5.16;
+
+import "hardhat/console.sol";
```

Comment:
remove this import

Reply:
Fixed in this [commit](https://github.com/VenusProtocol/venus-protocol/pull/210/commits/46f0cf168db69632d3f6a108f589965f5544c579)